core/src/test/java/io/questdb/test/griffin/engine/functions/bind/BindVariableServiceImplTest.java

```diff
@@ -750,6 +752,29 @@ public void testSetIntVarToShort() throws Exception {
         });
     }
 
+    @Test
+    public void testSetIntoUndefined() throws Exception {
```

Comment:
Binary column is not tested here. Is that on purpose?

Reply:
good catch, I postponed and then forgot, thanks!